packages/repo-tools/src/commands/api-reports/api-extractor.ts

```diff
@@ -363,7 +364,9 @@ export async function runApiExtraction({
   }
   const warnings = new Array<string>();
 
-  for (const { packageDir, name } of packageEntryPoints) {
+  for (const [packageDir, group] of Object.entries(
+    groupBy(packageEntryPoints, ep => ep.packageDir),
+  )) {
```

Comment:
this change of the main for loop could in theory be avoided, but it fits in with the new functionality and has the benfit that the command now no longer prints out the "Processing ${packageDir}" message multiple times when there are multiple exports.